string goal_type
geometry_msgs/PoseStamped pose_goal
jaco_manipulation/BoundingBox bounding_box
sensor_msgs/JointState joint_goal
---
---